Skrevet om datepciker til ts

pethel opened the pull request:
Skrevet om datepicker til ts. 

Det var gammel kode med event.wich som er deprecated. Det funket ikke ens i testverktøyen våre så fikk bytet ut det også. 

BREAKING CHANGE: default export -> named export

snek også in en opdatering av `@testing-library/user-event'` når jag provade funna ut vorfor event ikke funka som tilslut berodde på att dom var deprecated.

antidecaf reviewed (changes requested):
Ser ut som det er endringer i `ffe-account-selector-react`, `ffe-form-react` og `ffe-searchable-dropdown-react`, selv om commiten kun peker på `ffe-datepicker-react`. Skal disse være med? I så fall må de inn i egne commits for at Lerna skal kunne publisere de.

I [previewen](https://black-beach-0d62d0d03-1994.westeurope.2.azurestaticapps.net/) ser det ut til at `language`-propen ikke lenger funker som den skal. Sammenlignet eksemplene (Calendar-en, Datepicker-en osv) med [siste tilgjengelige component overview](https://sparebank1.github.io/designsystem/datepicker_Calendar_en), og der virker det fortsatt som det skal. Virker som det bare er å endre fra `language` til `locale` i eksemplene, men det bør kanskje også beskrives i bodyen til commit-meldinga.

Commit:
feat(ffe-datepicker-react): rewrite to ts

label and validation props are removed. use InputGroup and own validation.
default exports -> named exports

BREAKING CHANGE: label and validation props removed, default exports are now named exports

Commit:
test(ffe-account-selector-react): bump @testing-library/user-event version

Commit:
test(ffe-form-react): bump @testing-library/user-event version

Commit:
test(ffe-searchable-dropdown-react): bump @testing-library/user-event version

Commit:
test: bump @testing-library/user-event version

Commit:
docs(component-overview): update calendar and datepicker examples
